web/modules/custom/error_page/README.md

```diff
@@ -0,0 +1,132 @@
+# Description
+
+Shows a friendly page to the users when an exception or an error are thrown,
+instead of the white page with the error backtrace. The module is able to attach
```

Comment:
This is not entirely correct. The scope of the ticket was to have a way to associate a certain backtrace/error (logged with the appropriate logger) with a certain error message shown on the website.
At the moment, we have no way to associate a WSOD message with a certain error so either we properly configure the error messages level to show the full error and have the user report that, or we hide completely the errors (like should be in production) but we lose the tracking of the cause of the WSOD.
See my remark below for more on this.